### PR TITLE
Localstorage Keys Prefixed - issue:#4

### PR DESCRIPTION
**Pull request by rpc404**
Defined a constant variable in the library that holds the prefix value and used it throughout the library to prefix all the local storage keys to minimize the chances of conflicts and ensure that the library works reliably across different environments.

### CODE REVIEW

File: src/connect.js [hunk 1]
```diff
@@ -1,38 +1,38 @@
+import nacl from "tweetnacl";
```
Comment (frankhinek) [inline src/connect.js:1]:
```suggestion
import nacl from 'tweetnacl';
```

**Comment by frankhinek** [comment]
Hey @rpc404 -- thanks for submitting this PR to address the potential for local storage key conflicts.

A few asks before I can complete the review and merge:
- Remove formatting changes from README
- Switch all double-quotes back to single-quote -- convention for all of our libraries and you'll see the rule in the eslint file
- Remove trailing commas for last item in array/object
- Remove indentation changes for the switch statements
- Remove changes to line wrapping

Thanks

**Comment by rpc404** [comment]
Suggestion updated.

**Comment by michaelneale** [comment]
LGTM

**Comment by frankhinek** [comment]
Hey @rpc404 -- sincerely appreciate your contribution and follow-up.  This library is still in a pre-beta stage and there was a major refactoring that has been pending review & approval for the past week+.  With the merging of PR #13, a modular storage approach was introduced which both uses a "web5-" prefixed browser storage key and also enables the developer to use a custom storage option of their choosing.

As a result of the fairly significant refactor, the key collision issue was resolved.

The API should be stable going forward and it will be very unlikely for such a significant refactor to occur, so we'd welcome your questions on Discord and additional code contribution.

**Pull request closed without merging**